### PR TITLE
LPS-159023 [Raylife AP] Create layout of dashboard sales chart

### PR DESCRIPTION
**Pull request by joanagrego**
Related with --> [LPS-159023](https://issues.liferay.com/browse/LPS-159023)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @joanagrego

**Comment by pedro-michel** [comment]
ci:test:relevant

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js [hunk 1]
```diff
@@ -13,13 +13,20 @@
...
+import ClayIcon from '@clayui/icon';
 import React, {useEffect, useRef} from 'react';
 
+import ClayIconProvider from '../../context/ClayIconProvider';
+
 import './index.css';
 
 const DonutChart = ({
 	chartData,
+	dateUntilGoal,
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js:25]: Please, it needs to be a generic chart. Change ```dateUntilGoal```.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js [hunk 2]
```diff
@@ -13,13 +13,20 @@
...
+	goalValue,
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js:26]: Please, it needs to be a generic chart. Change ```goalValue```.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js [hunk 3]
```diff
@@ -13,13 +13,20 @@
...
+	hasLegend = false,
 	height = 190,
+	salesValue,
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js:29]: Please, it needs to be a generic chart. Change ```salesValue```.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
+.dashboard-sales-container {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.css:1]: Please, change the class's name.
```suggestion
.sales-dashboard-container {
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+				<ClaySelect
+					className="dashboard-sales-select"
+					onChange={({target}) => {
+						setSelectedFilterDate(target.value);
+					}}
+					sizing="sm"
+					value={selectedFilterDate}
+				>
+					{options.map((item) => (
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.js:126]: Please, change it.
```suggestion
					{options.map((option) => (
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions :fire:

**Comment by joanagrego** [comment]
Just starting reviewing :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js (hunk 2 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js:26]: I changed that by refactoring the code.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js (hunk 3 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js:29]: I changed that by refactoring the code.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.css (hunk 4 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.css:1]: Actually, we're using .`dashboard-sales-container` pattern.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js (hunk 1 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js:25]: I changed that by refactoring the code.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.js (hunk 5 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.js:126]: Done!

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:test:relevant

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**